gtsfm/utils/verification.py

```diff
@@ -86,38 +91,89 @@ def essential_to_fundamental_matrix(
     return np.linalg.inv(camera_intrinsics_i2.K().T) @ i2Ei1.matrix() @ np.linalg.inv(camera_intrinsics_i1.K())
 
 
-def compute_epipolar_distances(
-    normalized_coords_i1: np.ndarray, normalized_coords_i2: np.ndarray, i2Ei1: EssentialMatrix
+def compute_epipolar_distances_sed(
+    coordinates_i1: np.ndarray, coordinates_i2: np.ndarray, i2Fi1: np.ndarray
 ) -> Optional[np.ndarray]:
-    """Compute symmetric point-line epipolar distances between normalized coordinates of correspondences.
+    """Compute symmetric point-line epipolar distance between corresponding coordinates in two images. The SED is the
+    geometric point-line distance and is an over-estimate of the gold-standard reprojection error. The over-estimate
+    can reject correspondences which are actually inliers.
```

Comment:
can we rename this function as `compute_epipolar_distance_squared_sed()`?

Reply:
or just `compute_epipolar_distance_sq_sed()` for brevity

Reply:
sure